packages/zosfiles/__tests__/api/methods/unmount/Unmount.unit.test.ts

```diff
@@ -0,0 +1,88 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Unmount, ZosFilesMessages } from "../../../..";
+
+import { ZosmfRestClient } from "../../../../../rest";
+
+describe("Unmount", () => {
+    const dummySession: any = {};
+
+    describe("fs", () => {
+        const fileSystemName = "TEST.ZFS";
+
+        it("should succeed with correct parameters", async () => {
+            (ZosmfRestClient as any).putExpectString = jest.fn(() => {
+                // Do nothing
+            });
+            await Unmount.fs(dummySession, fileSystemName);
```

Comment:
Same comment as the mount unit tests... can we expect that the rest client is called?